File: 1.Basic-Concepts/b.structs/structs.c
#include <stdio.h>

struct Rectangle
{
    int length;
    int width;
    char x;
};

int main()
{
    struct  Rectangle rec = {100,50};
    struct  Rectangle rec2 = {.length=150, .width=300};

    printf("Char size : %d, Struct size : %d \n",sizeof(char),sizeof(rec));

    return 0;
}

File: 1.Basic-Concepts/b.structs/go/structs.go
package main

import "fmt"

type Rectangle struct{
	length int
	width int
}

func main(){
  rec := Rectangle{length: 25,width: 50}
  rec2 := Rectangle{50,100}

  // Accessing struct fields
  fmt.Printf("Rec 1 fields -> length : %d ,witdth : %d\n",rec.length,rec.width)
  
  fmt.Println("Rec : ",rec)
  fmt.Println("Rec2 : ",rec2)
}